src/locale/locales/id/messages.po

```diff
@@ -0,0 +1,2478 @@
+msgid "Could not load feed"
+msgstr "Tidak dapat memuat feed"
+
+#: src/view/screens/ProfileList.tsx:830
+msgid "Could not load list"
+msgstr "Tidak dapat memuat daftar"
+
+#: src/view/com/auth/HomeLoggedOutCTA.tsx:62
+#: src/view/com/auth/SplashScreen.tsx:46
+msgid "Create a new account"
+msgstr "Buat akun baru"
+
+#: src/view/com/auth/create/CreateAccount.tsx:120
+msgid "Create Account"
+msgstr "Buat Akun"
+
+#: src/view/com/auth/HomeLoggedOutCTA.tsx:54
+#: src/view/com/auth/SplashScreen.tsx:43
+msgid "Create new account"
+msgstr "Buat akun baru"
+
+#: src/view/screens/AppPasswords.tsx:248
+msgid "Created {0}"
+msgstr "Dibuat {0}"
+
+#: src/view/com/modals/ChangeHandle.tsx:387
+#: src/view/com/modals/ServerInput.tsx:102
+msgid "Custom domain"
+msgstr "Domain kustom"
+
+#: src/view/screens/Settings.tsx:615
+msgid "Danger Zone"
+msgstr "Zona Berbahaya"
+
+#: src/view/screens/Settings.tsx:622
+msgid "Delete account"
+msgstr "Hapus akun"
+
+#: src/view/com/modals/DeleteAccount.tsx:83
+msgid "Delete Account"
+msgstr "Hapus Akun"
+
+#: src/view/screens/AppPasswords.tsx:221
+#: src/view/screens/AppPasswords.tsx:241
+msgid "Delete app password"
+msgstr "Hapus kata sandi aplikasi"
+
+#: src/view/screens/ProfileList.tsx:351
+#: src/view/screens/ProfileList.tsx:411
+msgid "Delete List"
+msgstr "Hapus Daftar "
+
+#: src/view/com/modals/DeleteAccount.tsx:212
+msgid "Delete my account"
+msgstr "Hapus akun saya"
+
+#: src/view/screens/Settings.tsx:632
+msgid "Delete my account…"
+msgstr "Menghapus akun saya..."
+
+#: src/view/com/util/forms/PostDropdownBtn.tsx:214
+msgid "Delete post"
+msgstr "Hapus post"
+
+#: src/view/com/util/forms/PostDropdownBtn.tsx:218
+msgid "Delete this post?"
+msgstr "Hapus post ini?"
+
+#: src/view/com/post-thread/PostThread.tsx:242
+msgid "Deleted post."
+msgstr "Post dihapus."
+
+#: src/view/com/modals/CreateOrEditList.tsx:218
+#: src/view/com/modals/CreateOrEditList.tsx:234
+#: src/view/com/modals/EditProfile.tsx:197
+#: src/view/com/modals/EditProfile.tsx:209
+msgid "Description"
+msgstr "Deskripsi"
+
+#: src/view/com/auth/create/Step1.tsx:96
+msgid "Dev Server"
+msgstr "Server Dev"
+
+#: src/view/screens/Settings.tsx:637
+msgid "Developer Tools"
+msgstr "Alat Pengembang"
+
+#: src/view/com/composer/Composer.tsx:143
+msgid "Discard"
+msgstr "Buang"
+
+#: src/view/com/composer/Composer.tsx:137
+msgid "Discard draft"
+msgstr "Buang draf"
+
+#: src/view/screens/Moderation.tsx:207
+msgid "Discourage apps from showing my account to logged-out users"
+msgstr "Cegah aplikasi untuk menampilkan akun saya ke pengguna yang tidak login"
+
+#: src/view/screens/Feeds.tsx:405
+msgid "Discover new feeds"
+msgstr "Temukan feed baru"
+
+#: src/view/com/modals/EditProfile.tsx:191
+msgid "Display name"
+msgstr "Nama tampilan"
+
+#: src/view/com/modals/EditProfile.tsx:179
+msgid "Display Name"
+msgstr "Nama Tampilan"
+
+#: src/view/com/modals/ChangeHandle.tsx:485
+msgid "Domain verified!"
+msgstr "Domain diverifikasi!"
```

Comment:
I'm suggesting to change this from `diverifikasi` to `terverifikasi`.

Reply:
Has this change been implemented?